citi-onboarding/DACIn
# feature(footer):create the footer

## WHAT I DID
- I've created the footer section;
- I've make the footer responsive;

## HOW TO TEST
1. get in feature/footer if git checkout
2. analise and check the cods
3. do the merge
